subprojects/docs/src/samples/userguide/files/archives/groovy/build.gradle

```diff
@@ -14,6 +14,19 @@ task unpackFiles(type: Copy) {
 }
 // end::unpack-archive-example[]
 
+// tag::unpack-archive-subset-example[]
+task unpackLibsDirectory(type: Copy) {
+    from(zipTree("src/resources/thirdPartyResources.zip")) {
+        include "libs/**"
```

Comment:
Add an annotation explaining that this extracts a subtree, specifically only the directories and files under `libs` at the root of the ZIP file.

Reply:
Any reason not to use an annotation? I'd like us to move away from comments in the samples if possible.

Reply:
I didn't know about them. It was changed to use annotation.